### PR TITLE
Refactor/refactor branch

### PR DESCRIPTION
**Pull request by pkvarnwal**
- [x] I have reviewed the pull request myself. 
- [x] I confirm that code style is correct and follows CrownStack code style guide.
- [x] I have checked the code after running on Simulator/emulator/Device (which ever applicable).
- [x] I am convinced that I have not added new bugs. 
- [x] The UI behaviour is correct.

### CODE REVIEW

**Comment by ashutoshsingh33** [comment]
@aashishdhawan  Please review.

File: app/src/main/java/com/eqdepot/activities/BranchDetailActivity.java [hunk 1]
```diff
@@ -178,4 +186,12 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
             }
         }
     }
+
+    private void hideTelephone() {
+        if(mBranches.getTelephone().length() < 10) imageViewTelephone.setVisibility(View.GONE);
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/activities/BranchDetailActivity.java]: this logic is not correct. Please make sure that it is hidden only when it is empty or nil

File: app/src/main/java/com/eqdepot/activities/BranchDetailActivity.java [hunk 2]
```diff
@@ -178,4 +186,12 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
...
+    }
+
+    private void hideTollFree() {
+        if(mBranches.getToll().length() < 10) imageViewTollFree.setVisibility(View.GONE);
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/activities/BranchDetailActivity.java]: Same goes here. Please check above.

File: app/src/main/java/com/eqdepot/utils/ConnectionDetector.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+public class ConnectionDetector {
+
+    private Context mContext;
+
+    public ConnectionDetector(Context context) {
+        this.mContext = context;
+    }
+
+    public boolean isConnectingToInternet() {
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/utils/ConnectionDetector.java]: Please remove this.

File: app/src/main/java/com/eqdepot/activities/RegisterActivity.java [hunk 4]
```diff
@@ -37,15 +55,72 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
+
+        if (TextUtils.isEmpty(email)) {
+            SnackBarFactory.createSnackBar(this, linearLayoutRoot, getString(R.string.enter_emailId));
+            return false;
+        } else if (!StringUtils.isValidEmailId(email)) {
+            SnackBarFactory.createSnackBar(this, linearLayoutRoot, getString(R.string.invalid_email));
+            return false;
+        }
+
+        if (phoneNumber.length() < 1 || phoneNumber.length() < 10) {
```
Comment (aashishdhawan) [inline app/src/main/java/com/eqdepot/activities/RegisterActivity.java]: Please remove this.

File: app/src/main/java/com/eqdepot/activities/BranchDetailActivity.java (hunk 1 above)
Comment (pkvarnwal) [reply app/src/main/java/com/eqdepot/activities/BranchDetailActivity.java]: Okay

File: app/src/main/java/com/eqdepot/activities/BranchDetailActivity.java (hunk 2 above)
Comment (pkvarnwal) [reply app/src/main/java/com/eqdepot/activities/BranchDetailActivity.java]: Done

File: app/src/main/java/com/eqdepot/utils/ConnectionDetector.java (hunk 3 above)
Comment (pkvarnwal) [reply app/src/main/java/com/eqdepot/utils/ConnectionDetector.java]: Done

**Pull request merged**